docs/Keepers/overview.html

```diff
@@ -0,0 +1,74 @@
+
+### Funding
+The network is designed to perform your work even under conditions where gas prices spike. In order to ensure appropriate payment to the Keepers, your LINK balance must exceed the maximum amount that could be paid for performance. This maximum amount is calculated using [Chainlink’s on-chain fast gas feed](https://data.chain.link/ethereum/mainnet/gas/fast-gas-gwei) and a gas ceiling multiplier (see [configurations](#configurations)) that places a ceiling on payments to the Keeper.
+
+
+### Payments
+When a Keeper performs your Upkeep, the Keeper Registry will deduct from your Upkeep’s LINK balance and allocate it to the Keeper’s address. The amount deducted includes a premium on top of the transaction fee paid by the Keeper and uses our up to date price feeds for exchange rate conversion. See payment premium below.
+
+### Price Selection & Gas Bumping
+Chainlink Keeper nodes select the gas price dynamically based on the prices of transactions within the last several blocks, optimizing gas price based on current network conditions. Keepers are configured to select a price based on a target percentile.
+
+If the Keeper node does not see the `performUpkeep` transaction get confirmed within the next few blocks, it will automatically replace the transaction and bump the gas price. This will repeat until the transaction is confirmed.
+
+## Keepers Turn Taking
+Our Keepers take responsibility for your Upkeep in turns. The registered Upkeeps are broken into buckets based on the number of Keepers on the network, and at the end of each turn the buckets rotate from one Keeper to the next. So even if a Keeper goes down, we have built in redundancy and your Upkeep will be performed by the next one in line.
+
+The duration of a turn - the number of blocks a Keeper is responsible for a given bucket of Upkeeps - is configured within the KeeperRegistry. See below for the current block count per turn.
+
+Once a Keeper has performed an Upkeep, it cannot do so again until another Keeper on the network has subsequently performed the same Upkeep. This protects against a faulty or malicious Keeper from taking repeated action on a given Upkeep.
+
+## Configuration
+These configurations may change as we grow our users and the network as a whole.
+{% endmarkdown %}
+{% markdown %}
+
+* **Payment Premium (paymentPremiumPPB)** - <span id="show-paymentPremiumPPB"></span>
+    This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+* **Block Count Per Turn (blockCountPerTurn)** - <span id="show-blockCountPerTurn"></span>
+    The number of blocks in each turn before the Upkeep is rotated to the next Keeper
+* **Check Gas Limit (checkGasLimit)** - <span id="show-checkGasLimit"></span>
+    The maximum amount of gas that can be used by your checkUpkeep for off-chain computation.
+* **Call Gas Limit (callGasLimit)** - <span id="show-callGasLimit"></span> <!-- @TODO This doesn't seem to be a thing. -->
```

Comment:
This will be a thing when we do the registry migration. We can publish without this for now. 

For the time being, we an just hardcode the docs to say 2,500,000.